cicerone/app/src/main/java/com/teampower/cicerone/MainActivity.kt

```diff
@@ -0,0 +1,79 @@
+package com.teampower.cicerone
+
+import android.os.Build
+import android.os.Bundle
+import android.util.Log
+import android.view.Menu
+import android.view.MenuItem
+import androidx.annotation.RequiresApi
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.material.snackbar.Snackbar
+import kotlinx.android.synthetic.main.activity_scrolling.*
+
+const val MY_PERMISSIONS_REQUEST_LOCATION_ID = 99
+const val CHANNEL_ID = "CiceroneComms1337"
+const val TAG = "Cicerone"
+const val TAG_GEO = "Geofencer"
+
+class MainActivity : AppCompatActivity() {
+    private val latCon = LocationController()
```

Comment:
Really good job splitting up into different files 👍It makes it a lot easier to work with.